Sources/SwiftProcessing/Core/UIKitElements/Capture.swift

```diff
@@ -0,0 +1,310 @@
+import UIKit
+import AVFoundation
+import Vision
+
+open class Camera: UIKitViewElement {
```

Comment:
run the auto format on any file edited in this PR. Some tabbing seems off. 

In xCode, select all the code in the file, and hit control + i